help_function/src/onedpl_test_group_load.cpp

```diff
@@ -0,0 +1,152 @@
+// ====------ onedpl_test_group_load.cpp------------ *- C++ -* ----===//
+
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//
+// ===----------------------------------------------------------------------===//
+#include <sycl/sycl.hpp>
+#include <dpct/dpct.hpp>
+#include <dpct/dpl_utils.hpp>
+#include <iostream>
+#include <oneapi/dpl/iterator>
+
+bool test_load_blocked_striped(dpct::group::load_algorithm T) {

```

Comment:
Is this function testing only the "direct" version, only the "striped" version or both?  
It named specific to the striped version, but accepts a `load_algorithm`, and seems to verify the "direct" version only.

Please make sure these build run and pass when paired with your PR.

Reply:
Yes validated them on my local branch, should pass all tests now.  Now there is a branch for load_algorithm based tests.